src/mealAPI.js

```diff
@@ -0,0 +1,19 @@
+class ApiData {
+  static randomMeal = async () => {
+    const res = () => fetch('https://www.themealdb.com/api/json/v1/1/random.php')
+      .then((response) => response.json())
+      .then((json) => json);
+    const data = await res();
+    return data;
+  };
+
+  static getMeal = async (id) => {
+    const res = () => fetch(`https://www.themealdb.com/api/json/v1/1/lookup.php?i=${id}`)
+      .then((response) => response.json())
+      .then((json) => json);
```

Comment:
- [ ] <b>REQUIRED:</b> As requested in the previous review, you have to use `async/await` in case of any API request in this project. Kindly do that to comply with the project specifications.